src/components/page/FoodsPage.tsx

```diff
@@ -74,17 +77,17 @@ export const FoodsPage = ({ containers }: { containers: IContainer[] }) => {
       <div className="relative">
         <SearchBar />
         <FilterButton
-          containers={containersGroupedByGroupId}
-          containerIdMap={containerIdMap}
-          groupIdMap={groupIdMap}
+          isFilterSet={isFilterSet}
+          containerIdsGroupedByGroupId={containersGroupedByGroupId}
```

Comment:
How about rename like this?
`containerIdsGroupedByGroupId` -> `groupIdContainerIdsMap`

I don't think it is necessary to align the naming rules for the Map across the repository, but I thought it would be easier to understand if we align the naming rules to `{key}{value}Map` in the FilterButton props at least.

Reply:
Thanks for agreeing with my suggestion :)

You seem to be commenting on the older version of the git. Please refresh the code diff page, and you will see the updated state for the renaming.